packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2285,8 +2320,17 @@ mixin WidgetInspectorService {
   /// [WidgetInspectorService] can track which events were dispatched by
   /// overriding this method.
   @protected
-  void postEvent(String eventKind, Map<Object, Object?> eventData) {
-    developer.postEvent(eventKind, eventData);
+  void postEvent(String eventKind, Map<Object, Object?> eventData, {String stream = 'Extension'}) {
+    developer.postEvent(eventKind, eventData, stream: stream);
+  }
+  
+  /// All events dispatched by a [WidgetInspectorService] use this method
```

Comment:
both these comments should be tweaked to match the Dart style. Comments should have a first line followed by a body.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements